src/game/tuning.h

```diff
@@ -32,15 +59,32 @@ MACRO_TUNING_PARAM(ShotgunCurvature, shotgun_curvature, 1.25f)
 MACRO_TUNING_PARAM(ShotgunSpeed, shotgun_speed, 2750.0f)
 MACRO_TUNING_PARAM(ShotgunSpeeddiff, shotgun_speeddiff, 0.8f)
 MACRO_TUNING_PARAM(ShotgunLifetime, shotgun_lifetime, 0.20f)
+MACRO_TUNING_PARAM(ShotgunWaterResistance, shotgun_liq_resistance, 0.25f)
 
 MACRO_TUNING_PARAM(GrenadeCurvature, grenade_curvature, 7.0f)
 MACRO_TUNING_PARAM(GrenadeSpeed, grenade_speed, 1000.0f)
 MACRO_TUNING_PARAM(GrenadeLifetime, grenade_lifetime, 2.0f)
 
+MACRO_TUNING_PARAM(HarpoonSpeed, harpoon_speed, 2000.0f)
+MACRO_TUNING_PARAM(HarpoonCurvature, harpoon_curvature, 1.25f)
+MACRO_TUNING_PARAM(HarpoonDragSpeed, harpoon_drag_speed, 10.0f)
+MACRO_TUNING_PARAM(HarpoonEntityDragSpeed, harpoon_entity_drag_speed, 50.0f)
+MACRO_TUNING_PARAM(HarpoonEntityMinSpeed, harpoon_entity_min_speed, 30.f)
+MACRO_TUNING_PARAM(HarpoonFlagMinSpeed, harpoon_entity_flag_min_speed, 8.0f)
+MACRO_TUNING_PARAM(HarpoonMultiplier, harpoon_speed_mltpr, 5.0f)
+MACRO_TUNING_PARAM(HarpoonEntityMultiplier, harpoon_entity_speed_mltpr, 10.0f)
+MACRO_TUNING_PARAM(HarpoonFlagDragSpeed, harpoon_flag_drag_speed, 8.0f)
+MACRO_TUNING_PARAM(HarpoonPickupIgnoreRange, harpoon_pickup_ignore_range, 5.0f)
+MACRO_TUNING_PARAM(HarpoonReloadTime, harpoon_reload_time, 1.0f)
+MACRO_TUNING_PARAM(HarpoonDamage, harpoon_damage, 2.0f)
+MACRO_TUNING_PARAM(HarpoonDelay, harpoon_extra_dmg_delay, 1.5f)
+
 MACRO_TUNING_PARAM(LaserReach, laser_reach, 800.0f)
 MACRO_TUNING_PARAM(LaserBounceDelay, laser_bounce_delay, 150)
 MACRO_TUNING_PARAM(LaserBounceNum, laser_bounce_num, 1)
 MACRO_TUNING_PARAM(LaserBounceCost, laser_bounce_cost, 0)
+MACRO_TUNING_PARAM(LaserWaterReachMultiplier, laser_liq_reach, 1.5f)
+MACRO_TUNING_PARAM(LaserWaterDiffraction, laser_liq_diffraction, 1.33f)
```

Comment:
@Stiopa866 As I already mentioned this value is to low.

<details>
  <summary>Physics POV: </summary>
(AkTuAlLy) normal water refractive index is 1.333, this is however dependend on the wavelength. As you can see [here](https://en.wikipedia.org/wiki/Optical_properties_of_water_and_ice) blue light (~450 to 500 nm) has a reflective index of 1.336
</details>

<details>
  <summary>Game POV:</summary>
 This effect is not very big and if you want to make it a game mechanic you maybe want to push this up a little
</details>

Reply:
I'd love more feedback on whether to make the diffraction stronger from the gameplay perspective. Anyone seeing this, chip in!